trunk/SUAVE/Components/Energy/Converters/Combustor.py

```diff
@@ -50,18 +57,20 @@ def __defaults__(self):
         self.tag = 'Combustor'
         
         #-----setting the default values for the different components
-        self.fuel_data                      = SUAVE.Attributes.Propellants.Jet_A()
-        self.alphac                         = 0.0
-        self.turbine_inlet_temperature      = 1.0
-        self.inputs.stagnation_temperature  = 1.0
-        self.inputs.stagnation_pressure     = 1.0
-        self.outputs.stagnation_temperature = 1.0
-        self.outputs.stagnation_pressure    = 1.0
-        self.outputs.stagnation_enthalpy    = 1.0
-        self.outputs.fuel_to_air_ratio      = 1.0
-        self.fuel_data                      = Data()
-    
-    
+        self.fuel_data                       = SUAVE.Attributes.Propellants.Jet_A()
+        self.alphac                          = 0.0
+        self.turbine_inlet_temperature       = 1.0
+        self.inputs.stagnation_temperature   = 1.0
+        self.inputs.stagnation_pressure      = 1.0
+        self.inputs.static_pressure          = 1.0
+        self.inputs.mach_number              = 0.1
+        self.outputs.stagnation_temperature  = 1.0
+        self.outputs.stagnation_pressure     = 1.0
+        self.outputs.static_pressure         = 1.0
+        self.outputs.stagnation_enthalpy     = 1.0
+        self.outputs.fuel_to_air_ratio       = 1.0
+        self.fuel_data                       = Data()
+        self.area_ratio                      = 1.9
```

Comment:
This is a very specific area ratio to default to. Let's discuss this choice.

Reply:
Chosen at random. No data backing it up.